cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -558,9 +558,12 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         }
 
         note {
-          p("Note that no specific privileges are required for showing the current user's privileges using either `SHOW USER username PRIVILEGES`, or `SHOW USER PRIVILEGES`.")
```

Comment:
Does it make sense to have "Note that" followed immediately by another "please note". I'd have to look at the rendered version, but could they be combined?

Reply:
I think `note{ }` just gives you special formatting. Like in https://neo4j.com/docs/cypher-manual/4.0/syntax/lists/ that blue "I"